File: code_examples/3-Structured_Problem_Solving/1-Variables/Go/main.go
/**
 * Created by: Mr. Coxall
 * Created on: Sep 2020
 * This program shows declaring variables
 */

package main

import (
	"fmt"
)

func main() {
	// variable definition
	is_current := true              // bool
	age := 32                       // int
	area := 32.4                    // float32
	someWords := "Hello, World!"   // string

	fmt.Println(is_current)
	fmt.Println(age)
	fmt.Println(area)
	fmt.Println(someWords)

	fmt.Println("\nDone.")
}

File: code_examples/3-Structured_Problem_Solving/1-Variables/C/main.c
// Copyright (c) 2020 Mr. Coxall All rights reserved.
//
// Created by: Mr. Coxall
// Created on: Sep 2020
// This program shows declaring variables

#include <stdio.h>
#include <stdbool.h>

int main() {
    // variable definition
    bool isCurrent = true;  // bool
    int age = 42;  // int
    float area = 42.42;  // float
    char someWords[13] = "Hello, World!";  // string

    printf("%d\n", isCurrent);
    printf("%d\n", age);
    printf("%.2f\n", area);
    printf("%s\n", someWords);

    printf("\nDone.\n");
    return 0;
}
